### ISSUE TITLE
Different score values between chromosight detect and chromosight quantify on borders pattern

### ISSUE THREAD

**Issue by tfoutelrodier** [koszullab/chromosight#46]
Hello,
I detected borders pattern using chromosight detect and then wanted to quantify those borders scores across different conditions. However, when running chromosight quantify on the initial matrix, border scores were much lower than those found using chromosight detect (except for the first border detected). Is this result expected ?

Here are the command line I used (with Chromosight version 1.4.1): 
chromosight detect --pattern=borders --threads 6 --min-separation 20000 -p 0.5 -W 9 matrix.cool test/borders_detect
chromosight quantify -W 9 --pattern=borders test/borders_detect.tsv matrix.cool test/borders_quantify

Bellow are parts of the output files: 
Chromosight detect output:
chrom1	start1	end1	chrom2	start2	end2	bin1	bin2	kernel_id	iteration	score	pvalue	qvalue
chromosome_ref	1315000	1320000	chromosome_ref	1315000	1320000	263	263	0	0	0.8382847806	0.0000000000	0.0000000000
chromosome_ref	110000	115000	chromosome_ref	110000	115000	22	22	1	0	0.7745989939	0.0000000000	0.0000000001
chromosome_ref	205000	210000	chromosome_ref	205000	210000	41	41	1	0	0.7087168920	0.0000000331	0.0000000777
chromosome_ref	260000	265000	chromosome_ref	260000	265000	52	52	1	0	0.5554020576	0.0000748932	0.0000818600
chromosome_ref	505000	510000	chromosome_ref	505000	510000	101	101	1	0	0.5250500651	0.0001567472	0.0001601547
chromosome_ref	710000	715000	chromosome_ref	710000	715000	142	142	1	0	0.7006494326	0.0000000181	0.0000000448
chromosome_ref	780000	785000	chromosome_ref	780000	785000	156	156	1	0	0.5201083470	0.0001869179	0.0001869179
chromosome_ref	1030000	1035000	chromosome_ref	1030000	1035000	206	206	1	0	0.6466182488	0.0000006142	0.0000011547

Chromosight quantify output: 
chrom1	start1	end1	chrom2	start2	end2	bin1	bin2	score	pvalue	qvalue
chromosome_ref	1315000	1320000	chromosome_ref	1315000	1320000	263	263	0.8382847806	0.0000000000	0.0000000000
chromosome_ref	110000	115000	chromosome_ref	110000	115000	22	22	0.2937926314	0.0497858994	0.1559958182
chromosome_ref	205000	210000	chromosome_ref	205000	210000	41	41	0.3053649535	0.0488575488	0.1559958182
chromosome_ref	260000	265000	chromosome_ref	260000	265000	52	52	0.2115458335	0.1743224566	0.3724161573
chromosome_ref	505000	510000	chromosome_ref	505000	510000	101	101	-0.1663934145	0.2763862300	0.4811167707
chromosome_ref	710000	715000	chromosome_ref	710000	715000	142	142	0.2302008405	0.1287384415	0.3184582499
chromosome_ref	780000	785000	chromosome_ref	780000	785000	156	156	0.2003301451	0.1881451788	0.3770751398
chromosome_ref	1030000	1035000	chromosome_ref	1030000	1035000	206	206	-0.0722279965	0.6391357794	0.7952778302

**Comment by cmdoret** [comment]
Thanks for the bug report !

I managed to reproduce the issue and it happens only for the borders pattern.
This pattern uses 3 separate kernel and - should - return the best score of the 3 kernels.
The implementation was correct in `detect`, but quantify would always return the score of the first kernel, regardless of the score of the others.

I pushed a fix in 1cd64e1 and made a new release (1.5.0) available on pip. Feel free to close this issue if it fixes your problem.
